python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -322,8 +322,6 @@ def test_torch_tensor_custom_comm(ray_start_regular):
     sender = actor_cls.remote()
     receiver = actor_cls.remote()
 
-    from cupy.cuda import nccl
-
     class TestNcclGroup(GPUCommunicator):
```

Comment:
@stephanie-wang Rui moves `TestNcclGroup` back inside each test in the patch PR. This is not necessary though.